services/web/web-ui/src/docs/asciidoc/configuration.adoc

```diff
@@ -27,6 +27,7 @@ NONE
 |===
 |name|default|mandatory?|Description
 
+|operatorfabric.keycloak.realm||no|The realm name in keycloak server settings page
```

Comment:
```suggestion
|operatorfabric.keycloak.realm||yes|The realm name in keycloak server settings page. This is used for the log out process to know which realm should be affected.
```
This property is mandatory because log out won't work properly if it is absent.